browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchResult.java

```diff
@@ -0,0 +1,27 @@
+package com.braintreepayments.browserswitch;
+
+public class BrowserSwitchResult {
+    public static final int STATUS_OK = 1;
+    public static final int STATUS_CANCELLED = 2;
+    public static final int STATUS_ERROR = 3;
+
+    public int getStatus() {
+        return status;
+    }
+
+    public String getErrorMessage() {
+        return errorMessage;
+    }
+
+    private int status;
```

Comment:
I notice you've been putting private properties below public methods. Across the other Android SDKs we have both public and private properties at the top of the file. Are we looking to move away from this pattern?

Reply:
@scannillo Good call, that's an oversight. Right now looking to update BraintreeFragmentUnitTest to fit the new client model and then will do a second pass on formatting.